src/helm/benchmark/run_specs.py

```diff
@@ -2243,6 +2378,116 @@ def get_anthropic_hh_rlhf_spec(num_respondents: int, subset: str) -> RunSpec:
     )
 
 
+@run_spec_function("cleva")
+def get_cleva_spec(task: str, version: str, subtask: str = None, prompt_id: int = 0) -> RunSpec:
+    CLEVAScenario.download_dataset(version)
```

Comment:
Don't call `download_dataset` here. The downloading should happen inside `get_instances()` instead.

Reply:
`download_dataset` not only downloads training and testing data. It downloads prompt templates and inference parameters for each scenario/subtask as well. We need to obtain prompt templates and inference parameters during the construction of the `RunSpec`, before `get_instances` is called. 

Also because a scenario is not instantiated yet, we cannot really access the `self.output_path` when downloading prompt templates and inference parameters. We can download the data separately, but this solution may not be elegant either.

We are eager to learn your insights on solving this dilemma. We will take immediate action according to your guidance. Thank you!

Reply:
Thanks for the explanation. Let's merge this for now. I'll eventually open a new PR to provide another way of getting this path, so that we don't need to hardcode it.